src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -175,6 +177,25 @@ public SupportPlugin() {
         handler.setDirectory(getLogsDirectory(), "all");
     }
 
+    @Initializer(after = InitMilestone.EXTENSIONS_AUGMENTED)
+    public static void migrateExistingLogs() {
+        File rootDirectory = getRootDirectory();
+        File[] files = rootDirectory.listFiles();
+        if (rootDirectory != null) {
+            for (File f : files) {
+                if (f.isFile() && f.getName().endsWith(".log")) {
```

Comment:
Do we also move `*.zip`?

Reply:
No, they stay under `JENKINS_HOME/support/`